src/components/List.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { View, FlatList, Text, Image } from 'react-native';
+import Header from './Header';
+import ItemList from './ItemList';
+import addIcon from '../assets/icons/add.png';
+import Button from './Button';
+import styles from './List.styles';
+import strings from '../assets/strings';
+
+class List extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: props.items,
+    };
+  }
+
+  componentWillReceiveProps({ items }) {
```

Comment:
This method has been deprecated by React (see [here](https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops))

It's not clear to me why you're grabbing the props and setting the state with them each time. You should have no problem using the props just as they come. By setting the state here you might be triggering multiple renders.

Is this a thing required by the `FlatList` component?

Reply:
Yes its because I don't have redux yet, its a workaround for thing required by the FlatList

Reply:
Ok, then I would add a `TODO` or `FIXME` comment addressing this somewhere. Just so that if anyone looks at this they know why this is like it is.

Reply:
I'll keep it in mind in the future issues like that